packages/taro-components-advanced/src/components/water-flow/root.ts

```
@@ -0,0 +1,495 @@
+    if (this.lowerThresholdCount === 0) {
+      this.lowerThresholdScrollTop =
+        this.getState().scrollHeight - this.getState().containerSize.height
+      return 0
+    }
+    const sectionSize = this.sections.length
+    const tracker = Array.from(
+      { length: sectionSize },
+      () => new Map<number, { accCount: number, scrollTop: number }>()
+    )
+    // 从最后一个分组开始计算
+    loopSeciton: for (let i = sectionSize - 1; i > 0; i--) {
```

Biome (1.9.4)

[error] Unnecessary label.

Safe fix: Remove the unnecessary label.
You can achieve the same result without the label.


(lint/complexity/noUselessLabel)